code/modules/vore/fluffstuff/custom_clothes_vr.dm

```diff
@@ -2330,4 +2330,55 @@ Departamental Swimsuits, for general use
 	if(!istype(usr, /mob/living)) return
 	if(usr.stat) return
 
-	colorswap(usr)
\ No newline at end of file
+	colorswap(usr)
+
+//Pandora029 : Evelyn Tareen
+/obj/item/clothing/suit/storage/hooded/wintercoat/security/fluff/evelyn
+	name = "warden's navy winter coat"
+	desc = "A custom tailored security winter coat in navy blue colors, this one has the rank markings of a warden on it."
+	icon = 'icons/vore/custom_clothes_vr.dmi'
+	icon_state = "evelyncoat"
+
+	icon_override = 'icons/vore/custom_onmob_vr.dmi'
+	item_state = "evelyncoat_mob"
+
+	armor = list(melee = 25, bullet = 20, laser = 20, energy = 15, bomb = 20, bio = 0, rad = 0) //same as other sec coats
```

Comment:
This is redundant if it is a subtype of security coats. You can safely delete this line.